### DISCUSSION TITLE
Legal issues regarding proprietary_codecs in chromium (bundled with electron)

### DISCUSSION THREAD

**Discussion by SinghManmohan** [atom/atom#22527] [Q&A]
Hi,

We're in the process of building and deploying a desktop based app on electron v 12.0.7. It'll be a (free) commercial software deployed to ~2-3M users worldwide.
Recently our legal team enquired about the proprietary codecs bundled with chromium and I thought the best way to confirm this would be to reach out here.

So, here're specific things that I need your help with:
1. Are we shipping chromium with proprietary codecs? If yes, what are potentially contentious codecs that come by default in chromium bundled with electron v 12.0.7?
2. Is there any documentation around how the electron team builds chromium (with what flags)?
3. Is the default libffmpeg.dylib/dll bundled with electron building ffmpeg with support for things like H.264 and MP3 codecs?
4. I noticed there are libffmpeg binaries available with each electron build, e.g. https://github.com/electron/electron/releases/download/v12.0.7/ffmpeg-v12.0.7-darwin-arm64.zip . What's the purpose of this binary?

Any help would be appreciated.

Thanks & Regards
Manmohan Singh

**Comment by ThatXliner** [comment]
This is the issue tracker for Atom. If you have questions about electron or Atom, go to [GitHub Discussions](https://github.com/atom/atom/discussions).
